spec/file_test_runner_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+class BaseTestRunner < Mumukit::FileTestRunner
+  def run_test_command(path)
+    "cat #{path}"
+  end
+end
+
+class EmbeddedEnvTestRunner < BaseTestRunner
+  include Mumukit::WithEmbeddedEnvironment
+end
+
+class IsolatedEnvTestRunner < BaseTestRunner
+  include Mumukit::WithIsolatedEnvironment
+end
+
+Mumukit.configure do |c|
+  c.docker_image = 'ubuntu'
+end
+
+class File
+  def unlink
+  end
+end
+
+
+describe Mumukit::FileTestRunner do
```

Comment:
I don't understand why you are testing `run_compilation!` and not `run_test_file!`